lib/_http_client.js

```diff
@@ -311,7 +311,11 @@ ClientRequest.prototype._implicitHeader = function _implicitHeader() {
                     this[outHeadersKey]);
 };
 
-ClientRequest.prototype.abort = function abort() {
+ClientRequest.prototype.destroy = function destroy(error) {
+  if (this.res && this.res.complete) {
+    return;
```

Comment:
This does not make `destroy()` equivalent to `abort()` or am I missing something?

Reply:
Yes, you are right. Though it doesn't make sense to `abort` if already completed?

Reply:
It could be used to destroy the socket, even if complete. It seems you had to change a test exactly because of this.

Reply:
No, I changed a test for `destroy` (not `abort`) which was testing for what i believe is incorrect behaviour.

Reply:
Hm, eitherway. I think you have a point. I've updated.